segments/anaconda/anaconda.spec

```diff
@@ -13,7 +13,7 @@ function setUp() {
 function testAnacondaSegmentPrintsNothingIfNoAnacondaPathIsSet() {
   local P9K_CUSTOM_WORLD='echo world'
```

Comment:
What happens to these `P9K_CUSTOM_function` declarations now that you are using tags?

Reply:
Ideally nothing. The tags is just syntactic sugar. Instead of writing `custom_segment` you now write `segment::custom`. But the function still remains `P9K_CUSTOM_SEGMENT='echo segment'`..
Have to verify that custom segments still work though.. 🙄